test/test_helper.rb

```diff
@@ -83,6 +83,8 @@
 
 class User < ActiveRecord::Base
   include ULID::Rails
+  self.primary_key = :id
```

Comment:
If these lines are not necessary for all versions of Rails, should we add an `if RAILS_4_2` so they’re deleted when we remove 4.2 compatibility again?

Reply:
I would keep these lines because they highlight that it's up to the user to define what are the primary keys. weather you define them in schema or specify them in the model.

Reply:
We can remove them when we remove rails 4.2 support. Having them in this "older" version with other rails versions doesn't hurt.